src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -77,7 +77,8 @@ private function compileArray(Node\Expr\Array_ $arrayNode, CompilerContext $cont
      * Compile constant expressions
      *
      * @param Node\Expr\ConstFetch $constNode
-     * @return bool|null
+     * @return bool|mixed|null
+     * @throws \Roave\BetterReflection\NodeCompiler\Exception\UnableToCompileNode
```

Comment:
Just noticed in this code - this uses `constant()`, which may cause autoloading... new issue?